src/ppx/geometry.cpp

```diff
@@ -363,6 +409,36 @@ class VertexDataProcessorPositionPlanar : public VertexDataProcessorBase
         return n;
     }
 
+    virtual uint32_t AppendVertexData(Geometry* pGeom, const TriMeshVertexData16& vtx) override
+    {
+        const uint32_t n = AppendDataToVertexBuffer(pGeom, GetPositionBufferIndex(pGeom), vtx.position);
```

Comment:
can this code block within the function being templated instead of a copy of existing code?

Reply:
Updated.